src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1173,7 +1203,7 @@ class VMDeviceService(CRUDService):
             Str('type', enum=['AHCI', 'VIRTIO'], default='AHCI'),
             Bool('exists', default=True),
             Bool('boot', default=False),
-            Int('size', default=0),
+            Int('size', default=1073741824),
```

Comment:
Why put a value, and why 1073741824?

Reply:
I defaulted  it to 1 GB but it does make more sense to have a null value as a default and only update this field if explicitly specified.

Reply:
Constants in the form of `1024*1024*1024` or `1024**3` make it clearer what the value is meant to be. Or declare a gigabyte constant and use it alone or `1*gigabyte`.
 Most readers can't easily recognize 10-digit numbers.